API/RestSharpAPI/Utilities/SystemConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Utilities
+{
+    public class SystemConfiguration
```

Comment:
I have a feedback on the way we are implementing Configuration settings per environment. Here we have a variable for each environment, for example DevelopmentConfigurationSettings, AcceptanceConfigurationSettings. 
Generally, we only have one class for all Configuration settings, and we only change the things that differ in each environment for example Urls. May be we can discuss this offline. I am briging this because this is a new project and we can implemnt things in correct way

Reply:
Yes, that is something that I want fixed.  I've had to set up this up as an independent project, when it should work as one with the UI run.  All the testing, logging and reporting should be one.